pulser-core/pulser/devices/_device_datacls.py

```diff
@@ -828,6 +890,33 @@ def from_abstract_repr(obj_str: str) -> Device:
             )
         return device
 
+    def _layout_lines(self) -> str:
+
+        layout_lines = [
+            "\nLayout parameters:",
+            f" - Requires layout: {self._param_yes_no(self.requires_layout)}",
+            " - Accepts new layout: "
+            + self._param_yes_no(self.accepts_new_layouts),
+            f" - Minimal number of traps: {self.min_layout_traps}",
+            self._param_check_none(self.max_layout_traps)(
+                " - Maximal number of traps: {}"
+            ),
+            f" - Maximum layout filling fraction: {self.max_layout_filling}",
+        ]
+
+        return "\n".join(line for line in layout_lines if line != "")
```

Comment:
I had something slightly different in mind for the layout parameters. I think the ones that are in `VirtualDevice` should also appear in its specs, so this method could be moved to `BaseDevice` _expect_ for the `accepts_new_layout` line. That one, we could add it like this:
```suggestion
    def _layout_lines(self) -> list[str]:
        layout_lines = super()._layout_lines()
        layout_lines.insert(2, f" - Accepts new layout: {self._param_yes_no(self.accepts_new_layouts)}" 
        return layout_lines
```